pkgs/development/libraries/fmt/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, cmake }:
+
+let
+  version = "5.2.1";
+in
+
+stdenv.mkDerivation rec {
+  name = "fmt-${version}";
+  src = fetchurl {
+    url = "https://github.com/fmtlib/fmt/archive/${version}.tar.gz";
+    sha256 = "3c812a18e9f72a88631ab4732a97ce9ef5bcbefb3235e9fd465f059ba204359b";
+  };
+  nativeBuildInputs = [ cmake ];
+  cmakeFlags = [ "-DFMT_TEST=no"
```

Comment:
Why tests have to be disabled?

Reply:
Checks were failing due to the tests not finding the shared object. Changed that by setting LD_LIBRARY_PATH in the preCheckHook. If this is problem is normally solved otherwise, please let me know.